projects/storefrontlib/src/lib/storefront.module.ts

```diff
@@ -13,7 +13,6 @@ import { StorefrontModuleConfig } from './storefront-config';
 
 import { CmsLibModule } from './cms-lib/index';
 import { CmsModule } from './cms/index';
-import { CxApiModule } from './cx-api/cx-api.module';
```

Comment:
aren't we using this module in the storefront?

Reply:
if so, please leave this import and the usage below, if not just ignore this comment

Reply:
That's module I moved from storefront to core :) It doesn't exist anymore there